problems/removeEvenStrings.js

```diff
@@ -6,8 +6,17 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+const removeEvenStrings = (arr) => {
+    let odds = []
+    for (let i = 0; i < arr.length; i++) {
+        const el = arr[i];
+        if (el.length % 2 === 1) {
+            odds.push(el)
+        } else if (arr.length === 0) {
```

Comment:
arr.length could never be 0 here, otherwise we wouldn't have entered the for loop.